### PR TITLE
:book:  Replace quickstart document by a link

### PR DESCRIPTION
**Pull request by jklippel**
Currently there are two quickstart documents.
This is very confusing. There should only be
one quickstart.

This commit replaces the quickstart document in
this repository with a link to the current quickstart document in the cluster-stacks repository.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -1,141 +1,5 @@
 # Quickstart
 
-This section guides you through all the necessary steps to create a workload Kubernetes cluster on top of the OpenStack infrastructure. The guide describes a path that utilizes the [clusterctl] CLI tool to manage the lifecycle of a [CAPI] management cluster and employs [kind] to create a local non-production management cluster.
+This document has moved.
```
Comment (michal-gubricky) [inline docs/quickstart.md:3]:
```suggestion
This document has been moved.
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -1,141 +1,5 @@
...
-[Docker]: https://www.docker.com/
-[kind]: https://kind.sigs.k8s.io/
-[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
-[clusterctl]: https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl
-[CAPO]: https://github.com/kubernetes-sigs/cluster-api-provider-openstack
-[CAPI]: https://cluster-api.sigs.k8s.io/
-[go]: https://go.dev/doc/install
-[envsubst]: https://github.com/drone/envsubst
-[jq]: https://jqlang.github.io/jq/download/
+You can find the current version at <https://github.com/SovereignCloudStack/cluster-stacks/blob/main/providers/openstack/README.md>.
```
Comment (michal-gubricky) [inline docs/quickstart.md:5]:
```suggestion
You can find the current version of the quickstart guide [here](https://github.com/SovereignCloudStack/cluster-stacks/blob/main/providers/openstack/README.md).
```

**Review by michal-gubricky — CHANGES_REQUESTED** [review]

**Comment by jschoone** [comment]
Hi @michal-gubricky, do you have to do anything here or is it a bug in GitHub? Because I can't see any unresolved change requests

**Comment by michal-gubricky** [comment]
> Hi @michal-gubricky, do you have to do anything here or is it a bug in GitHub? Because I can't see any unresolved change requests

Everything seems to be resolved from my side now after you have applied my second suggestion as well.

**Review by michal-gubricky — APPROVED** [review]

**Pull request merged**